src/app/pages/system/alertservice/alert-service/alert-service.component.ts

```diff
@@ -597,6 +600,37 @@ export class AlertServiceComponent {
           }],
           value: 'public',
         },
+        // Telegram
+        {
+          type: 'input',
+          name: 'Telegram-bot_token',
+          placeholder: helptext.Telegram_bot_token_placeholder,
+          tooltip: helptext.Telegram_bot_token_tooltip,
+          relation: [{
+            action: "SHOW",
+            when: [{
+              name: "type",
+              value: 'Telegram',
+            }]
+          }],
+          required: true,
+          validation: [Validators.required],
+        },
+        {
+          type: 'input',
```

Comment:
I notice you mention that this field takes a space/comma separated list of ids, we have another component type, the 'chip' that is preferable for these